build.gradle

```diff
@@ -34,19 +32,20 @@ repositories {
 }
 
 dependencies {
-    compile 'org.codehaus.groovy:groovy-all:2.4.7'
-    compile 'org.apache.ant:ant:1.9.4'
+    compile 'org.codehaus.groovy:groovy-all:2.5.13'
+    compile 'org.apache.ant:ant:1.10.8'
     compile 'org.codehaus.groovy.modules.http-builder:http-builder:0.7.1'
-    compile 'org.slf4j:slf4j-api:1.7.25'
+    compile 'org.slf4j:slf4j-api:1.7.30'
 
-    compileOnly 'junit:junit:4.12'
-    compileOnly 'org.junit.jupiter:junit-jupiter-api:5.3.2'
+    compileOnly 'junit:junit:4.13'
```

Comment:
`junit:junit` could be provided by `junit-vintage-engine`, but we can also have handle that in another PR.